benchmark/src/main/scala/keycloak/scenario/admin/CreateGroups.scala

```diff
@@ -0,0 +1,14 @@
+package keycloak.scenario.admin
+
+import keycloak.scenario.{CommonSimulation, KeycloakScenarioBuilder}
+
+
+class CreateGroups extends CommonSimulation {
+
+  setUp("CRUD Groups", new KeycloakScenarioBuilder()
+    .serviceAccountToken()
+    //.listGroups()
+    //.createGroups()
+    //.deleteGroups()
+  )
+}
```

Comment:
IMHO this file should not be part of this PR (yet).

Reply:
true, it slipped through my radar, will remove it